### DISCUSSION TITLE
How to use Qiskit Aer simulators instead of cusvaer?

### DISCUSSION THREAD

**Discussion by xumingkuan** [NVIDIA/cuQuantum#126] [Q&A] [answered]
According to https://docs.nvidia.com/cuda/cuquantum/latest/appliance/qiskit.html, an option `cusvaer_enable` has been added since cuQuantum Appliance 22.11. I want to compare Qiskit Aer simulators with cusvaer so I want to set `cusvaer_enable=False` to run Qiskit Aer simulators. But I got `backend: cusvaer_simulator_statevector` from the output of the following Python program:

```python3
from qiskit import Aer, QuantumCircuit, transpile

circuit = ...
simulator = Aer.get_backend('aer_simulator_statevector', cusvaer_enable=False)
simulator.set_options(**options)
circuit = transpile(circuit, simulator)
job = simulator.run(circuit)
result = job.result()
print(f'backend: {result.backend_name}')
```

I was using the container from https://github.com/NVIDIA/cuQuantum/discussions/36#discussioncomment-6481363 with cuQuantum Appliance 23.03 on Perlmutter. I wondered if this was because Qiskit only supports a single node, but I got `backend: cusvaer_simulator_statevector` no matter if I use 2 nodes with 4 GPUs each, 1 node with 4 GPUs, or 1 node with 1 GPU. Should I not use the container if I want to use Qiskit Aer simulators?

**Comment by ymagchi** [comment, marked as answer]
Hi @xumingkuan,

Could you please try passing the `cusvaer_enable` option at `simulator.set_options()`? It will disable the cusvaer backend and invoke Aer backends instead.

**Reply by xumingkuan** [reply]
This works, thanks!
